Client/src/component/Announcements/AnnouncementStore.tsx

```diff
@@ -1,20 +1,57 @@
-import {create} from 'zustand';
+// import {create} from 'zustand';
 
-interface AnnouncementStore {
-    title: string;
-    description: string;
-    setTitle: (title: string) => void;
-    setDescription: (description: string) => void;
-    resetForm: () => void;
-  }
+// interface AnnouncementStore {
+//     title: string;
+//     description: string;
+//     setTitle: (title: string) => void;
+//     setDescription: (description: string) => void;
+//     resetForm: () => void;
+//   }
   
-  const useAnnouncementStore = create<AnnouncementStore>((set) => ({
+//   const useAnnouncementStore = create<AnnouncementStore>((set) => ({
```

Comment:
You can remove commented code, later if you need to refer the code later you can refer the previous commits.